_includes/manuals/1.12/5.8_multiple_foremen.md

```diff
@@ -0,0 +1,4 @@
+
+The following steps are suggested when configuring multiple Foreman instances to work together. They will ensure that data, passwords, and cookies are shared between multiple instances.
+
+__$app_root__ is wherever you installed Foreman, usually __/usr/share/foreman__.
```

Comment:
Prefer using backticks (`) for highlighting paths etc, it uses code style formatting.